contracts/btc-staking/src/validation/mod.rs

```diff
@@ -241,3 +255,107 @@ pub fn verify_active_delegation(
 
     Ok(())
 }
+
+pub fn verify_undelegation(
```

Comment:
Not sure about this, but perhaps this can be a method of `BtcDelegation`. Problem is that that will require extending the feature flag to the package.

Reply:
The verification of an early unbonded BTC delegation is dfferent from verifying the undelegation info inside `BTCDelegation`. The former verifies whether the staker's signature over unbonding tx is valid. The latter verifies whether the staker provides correct unbonding tx, unbonding slashing tx, and signs relevant signatures correctly.